package.json

```
@@ -0,0 +1,46 @@
+{
+  "name": "permanent-sdk",
+  "version": "0.0.0",
+  "description": "Functional Node.js SDK for Permanent.org",
+  "main": "src/index.ts",
+  "scripts": {
+    "lint": "eslint ./src --ext .ts,.tsx",
```

We're probably not going to write any React in this repo!
```suggestion
    "lint": "eslint ./src --ext .ts",
```